NC_POST_BUILD_SCRIPT.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e # Exit (and fail) immediately if any command in this scriptfails
+
+# Check that our APK files are not bigger than they should be
+python tools/metrics/apk_size.py
+
+# buddybuild modifies our buildscripts and sources (this is partly to enable
+# their SDK, and partly to allow selecting flavours in the BuddyBuild UI).
+# We don't know where the Buddybuild SDK lives, which causes gradle builds
+# to be broken (due to the SDK dependency injected into FocusApplication),
+# it's easiest just to revert to a clean source state here:
+git reset --hard
```

Comment:
We don't need that reset any more since we have no Buddybuild SDK now.
Also maybe we'd like to remove the comments and ./gradlew jacocoTestReport.